### PR TITLE
Bybit: remove swap parameter in doc

### PR DESCRIPTION
**Pull request by phips28**
https://bybit-exchange.github.io/docs/v5/order/execution

image

### CODE REVIEW

File: ts/src/bybit.ts [hunk 1]
```diff
@@ -5082,12 +5082,12 @@ export default class bybit extends Exchange {
          * @method
          * @name bybit#fetchMyTrades
          * @description fetch all trades made by the user
-         * @see https://bybit-exchange.github.io/docs/api-explorer/v5/position/execution
+         * @see https://bybit-exchange.github.io/docs/v5/order/execution
          * @param {string} symbol unified market symbol
          * @param {int} [since] the earliest time in ms to fetch trades for
          * @param {int} [limit] the maximum number of trades structures to retrieve
          * @param {object} [params] extra parameters specific to the exchange API endpoint
-         * @param {string} [params.type] market type, ['swap', 'option', 'spot']
```
Comment (carlosmiei) [inline ts/src/bybit.ts:5090]: Why did you remove spot from here?

Comment (phips28) [reply ts/src/bybit.ts:5090]: I removed "swap" - spot is still there.

Comment (phips28) [reply ts/src/bybit.ts:5090]: I switched the order so that github does not mark it as removed.

Comment (carlosmiei) [reply ts/src/bybit.ts:5090]: Sorry my bad, but the question remains 😅  that endpoint supports swap markets as well

Comment (phips28) [reply ts/src/bybit.ts:5090]: I posted the link/screenshot from their api doc: https://bybit-exchange.github.io/docs/v5/order/execution

But I think I got confused, because CCXT has its own weird logic to inject the correct `category` with `type` & `subType`. 

Here: 

```
    getBybitType(method, market, params = {}) {
        let type = undefined;
        [type, params] = this.handleMarketTypeAndParams(method, market, params);
        let subType = undefined;
        [subType, params] = this.handleSubTypeAndParams(method, market, params);
        if (type === 'option' || type === 'spot') {
            return [type, params];
        }
        return [subType, params];
    }
```
